Fix for Hugo v0.146.0

medaglia opened the pull request:
In v0.146.0, hugo changed the dir structure in layouts, renaming partials to _partials, breaking this theme. This fixes issues like the following:

```
execute of template failed: template: index.html:15:15: executing "index.html" at <partial $partial $page>: error calling partial: partial "partials/sections/gallery.html" not found
```

medaglia committed:
Fix for Hugo v0.146.0

In v0.146.0, hugo changed the dir structure in layouts, renaming
partials to _partials, breaking this theme. This fixes those issues.

frothm commented on layouts/index.html:
I think you missed one more appearance of the old path (line 8)

medaglia replied:
Thanks. I've fixed that, as well as another issue I noticed from the version upgrade. And updated some docs as well.

medaglia committed:
docs: update references to use _partials and _shortcodes directories

Update documentation and template paths to reflect Hugo v0.146.0's new directory structure, where `partials` and `shortcodes` are now prefixed with underscores. Move shortcode files to `_shortcodes` directory and update all references in documentation and templates.
